zerver/webhooks/grafana/view.py

```diff
@@ -33,8 +37,24 @@ def api_grafana_webhook(
     if payload.get("message") is not None:
         message_text = payload["message"] + "\n\n"
 
+    alert_status = ""
```

Comment:
@AdamVB this shouldn't be required I think. Test it without and if its fine, then this looks good to me. Thanks for working on this.